app/ADAKOR/ADAKOR_task3.c

```diff
@@ -0,0 +1,75 @@
+/**
+ * Responsible: Wojciech Fiderek
+ *
+ *
+ * History log:
+ * 2017.12.06   ADAKOR  Initial version
+ */
+
+#include "ADAKOR_task3.h"
+#include "stdint.h"
+
+/* Every public function in your module should start with   "MODULENAME_"   prefix */
+/* Every private function in your module should start with  "MODULENAME__"  prefix */
+
+static uint8_t arru8Storage [(LAMP_COUNT % 8) ? LAMP_COUNT / 8 + 1 : LAMP_COUNT / 8];
+
+eErr_t ADAKOR_lampOn(eLamp_t eLamp)
+{
+	eErr_t eResult;
+	if (eLamp >= LAMP_START && eLamp <= LAMP_COUNT)
+	{
+		arru8Storage[eLamp / 8] |= (1 << eLamp % 8);
```

Comment:
Priority of % is higher than <<, so everything will be ok, but adding additional brackets would be nice